python/res/job_queue/ert_script.py

```diff
@@ -19,6 +19,28 @@ def __init__(self, ert):
 
         self.__is_cancelled = False
         self.__failed = False
+        self.stdout = ""
+        self.stderr = ""
+
+    @property
+    def stdout(self):
+        """ @rtype: str """
+        return self.__stdout
+
+    @stdout.setter
+    def stdout(self, value):
+        """ @type: str """
+        self.__stdout = value
+
+    @property
+    def stderr(self):
+        """ @rtype: str """
+        return self.__stdout
+
+    @stderr.setter
```

Comment:
Having public read&write properties with names `stdout` and `stderr` is in my opinion a bit misleading, it is not `stdout` and `stderr`as such you are manipulating here after all, but rather the string content which has been written to the respective streams.

Reply:
yeh, agreed.

Did a quick test on how it feels when using the existing `defaultStackTrace` in `ErtScript`. That might be the most "how the rest works", but I think `ErtScript` still should keep the results and have it available.